tools/sphere.c

```diff
@@ -77,11 +88,6 @@ int do_sloop( string arg ) {
         add_sloopee_shadow( sloopee );
         return 1;
     }
-    if ( sizeof( find_slooper_shadow( slooper ) ) && sizeof( find_sloopee_shadow( sloopee ) ) ) {
-        ansi_tell( slooper, sprintf("You are already slooping %s"
-            ,FINDP( arg )->query_name() ), BOLD_PURPLE );
-        return 1;
-    }
     ansi_tell( slooper, "Wubba Lubba Dub Dub! It's a sloop-a-doop, "
```

Comment:
Try and consolidate this logic with the block above. You may not even need to create a new function, what about:
```
if (something) {
  common stuff;
  if (something else) {
    add_slooper_shadow(slooper);
  }
}
```